selectors.yml

```diff
@@ -373,4 +373,16 @@ selectors:
     name: "private-testnet-mica"
   48900:
     selector: 17198166215261833993
-    name: "ethereum-mainnet-zircuit-1"
\ No newline at end of file
+    name: "ethereum-mainnet-zircuit-1"
+  223:
+    selector: 5406759801798337480
+    name: "bitcoin-mainnet-bsquared-1"
+  200901:
+    selector: 7937294810946806131
+    name: "bitcoin-mainnet-bitlayer-1"
+  60808:
+    selector: 3849287863852499584
+    name: "ethereum-mainnet-sepolia-bob-1"
```

Comment:
`mainnet-sepolia`? what's this?
If bob-1 is a rollup, and this is the mainnet, and anchored to ethereum-mainnet (i.e. homestead), this shoud be named `ethereum-mainnet-bob-1`. If it's a testnet anchored to ethereum-testnet-sepolia, then it should be named as such; mainnet-sepolia makes no sense in any case.

Reply:
Good catch, missed that one, I think that's a copy pasting error. We should remove sepolia

Reply:
Indeed, good catch Andre. Fixed it.